etc/testing/circle/workloads/pulumi/aws/app.go

```diff
@@ -24,6 +24,10 @@ func DeployApp(ctx *pulumi.Context, k8sProvider *kubernetes.Provider, saRole *ia
 	if enterpriseKey == "" {
 		return errors.WithStack(fmt.Errorf("need to supply env var ENT_ACT_CODE"))
 	}
+	detdockerconfigjson := os.Getenv("DOCKERCONFIGJSON")
```

Comment:
Are we able to change the environment variable to something explicitly determined like `DETDOCKERCONFIGJSON` or is this something we are stuck inheriting?

Reply:
👍 Yea let's do that. That way in circle we will know it's their docker repo.